pleskdistup/actions/emails.py

```diff
@@ -52,3 +54,31 @@ def _revert_action(self) -> action.ActionResult:
 
     def estimate_post_time(self) -> int:
         return 5
+
+
+class RestoreDovecotConfiguration(action.ActiveAction):
+    dovecot_config_path: str
+
+    def __init__(self):
+        self.name = "restore Dovecot configuration"
+        self.dovecot_config_path = "/etc/dovecot/dovecot.conf"
+
+    def _is_required(self) -> bool:
+        return os.path.exists(self.dovecot_config_path)
+
+    def _prepare_action(self) -> action.ActionResult:
+        files.backup_file(self.dovecot_config_path)
+        return action.ActionResult()
+
+    def _post_action(self) -> action.ActionResult:
+        path_to_backup = plesk.CONVERTER_TEMP_DIRECTORY + "/dovecot.conf.bak"
```

Comment:
What is `plesk.CONVERTER_TEMP_DIRECTORY`? Can't find it's definition.
Also, please use `os.path.join()`.

Reply:
> What is plesk.CONVERTER_TEMP_DIRECTORY?

It should be passed as argument, actually it is state_dir.